Chatkit/Authentication/Token Providers/TestTokenProvider.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import PusherPlatform
+
+/// TestTokenProvider retrieves tokens from the Chatkit service's Test Token Provider, which
+/// is for development user only, and must be enabled for your instance in the Chatkit Dashboard.
+///
+/// The test token provider will always sign a token for the requested userID, without applying any
+/// form of authentication.
+public class TestTokenProvider: TokenProvider {
+    
+    // MARK: - Properties
+    
+    private static let userIdentifierQueryItemName = "user_id"
+    private static let urlScheme = "https"
+    private static let urlHost = "pusherplatform.io"
+    private static let urlService = "services/chatkit_token_provider"
+    private static let urlResource = "token"
+    
+    /// The locator for your instance, the same value from the Dashboard which you use to construct
+    /// the Chatkit object.
+    public let instanceLocator: String
+    
+    /// The userID to fetch tokens for. A token will always be signed for this userID without any
+    /// authentication being applied.
+    public let userID: String
```

Comment:
`userIdentifier` would be my preference as well, but `userID` was defined in the draft of the public interface. I am happy to change that. @mdpye

Reply:
Yeah, go with "identifier", as used everywhere else in the SDK. It's a peculiarity of the iOS ecosystem, my fingers type "id" without we asking them to.

Although is android it's `userId`. Fun fact, in Go it's `userID` or the (non-configurable) linter will reject your code, because the authors don't understand the different between an abbreviation and an acronym...

By which rambling, I mean to say that any interface I propose ("specify") should be filtered for language bias, idiom and confusion :)

Reply:
I personally don't care whether we use `ID` `Id` or `Identifier`.  I just want to be consistent :-)